XamlControlsGallery/ControlPages/NavigationViewPage.xaml.cs

```diff
@@ -278,22 +296,50 @@ private void paneFooterCheck_Click(object sender, RoutedEventArgs e)
 
         private void panePositionLeft_Checked(object sender, RoutedEventArgs e)
         {
-            if ((sender as RadioButton).IsChecked == true && nvSample != null)
+            if ((sender as RadioButton).IsChecked == true)
             {
-                nvSample.PaneDisplayMode = Microsoft.UI.Xaml.Controls.NavigationViewPaneDisplayMode.Left;
-                nvSample.IsPaneOpen = true;
-                FooterStackPanel.Orientation = Orientation.Vertical;
+                if ((sender as RadioButton).Name == "nvSampleLeft" && nvSample != null)
```

Comment:
This if statement seems to be mostly the same code copy-pasted. Can it be refactored into something cleaner?

Reply:
I cleaned up this function so it added functionality for my sample (it was originally created to only be an event handler for radio buttons on another sample). Do you recommend that I combine all of the left/top/leftcompact pane clicked functions into one and just have a bunch of if statements?

Reply:
Gotcha. I just saw some duplicated code and was curious if we could optimize it. I'd prefer this than a bunch of if statements.